docs/installation.md

```diff
@@ -2,310 +2,156 @@
 
 ### Minimum requirements
 
-We have tested PlantCV on the following systems:
+PlantCV has been tested on the following systems:
 
-- Linux: CentOS 7 x86 64-bit (RedHat Enterprise Linux)
+- Linux: CentOS 7 (RedHat Enterprise Linux)
 - Linux: Ubuntu 12.04, 14.04, and 16.04
-- Mac OSX 10.11
+- Linux: Raspbian "Jessie"
+- Mac OSX 10.11 and macOS 10.12+
 - Windows 10
+- Cloud9 IDE
 
-A list of minimum tested software dependencies is listed below:
+#### Required dependencies
 
-- Python 2.7
-    - argparse 1.1
-    - cv2 2.4
-    - matplotlib 1.5
-    - numpy 1.11
-    - pandas 0.18
-    - pytest 2.9
-    - python-dateutil 2.6
-    - scikit-image 0.12
-    - scipy 0.18
-    - setuptools 28.6
-- OpenCV 2.4
-- SQLite 3.7
-- Git 1.8
+- Python (tested with versions 2.7 and 3.6)
+    - argparse
+    - cv2 (we recommend 2.14+ or 3.3+)
```

Comment:
The `cv2` version was min 2.4 before, and now 2.14. Is that ok?

Reply:
Good catch, I confused the OpenCV major and minor versions. I corrected it to 2.4.14.